schema.json

```diff
@@ -1775,6 +1775,22 @@
             "type": "array",
             "uniqueItems": true
         },
+        "contributors": {
+            "description": "Persons or entities who played a secundary role to the generation of the software or dataset.",
```

Comment:
```suggestion
            "description": "The contributor(s) to a work.",
```

**suggestion:** Word this in analogy with `authors`.

Reply:
Fine for me, if the others agree.

Reply:
OK with me!